src/nft/components/explore/Table.tsx

```diff
@@ -0,0 +1,112 @@
+import clsx from 'clsx'
+import { Column, IdType, Row, useSortBy, useTable } from 'react-table'
+
+import { Box } from '../../components/Box'
+import { CollectionTableColumn } from '../../types'
+import { ArrowRightIcon } from '../icons'
+import { ColumnHeaders } from './CollectionTable'
+import * as styles from './Explore.css'
+
+interface TableProps<D extends Record<string, unknown>> {
+  columns: Column<CollectionTableColumn>[]
+  data: CollectionTableColumn[]
+  hiddenColumns: IdType<D>[]
+  classNames?: {
+    td: string
+  }
+  getTrProps: (row: Row<CollectionTableColumn>) => Record<string, unknown>
+}
+
+export function Table<D extends Record<string, unknown>>({
+  columns,
+  data,
+  hiddenColumns,
+  classNames,
+  getTrProps,
+  ...props
+}: TableProps<D>) {
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } = useTable(
+    {
+      columns,
+      data,
+      initialState: {
+        sortBy: [
+          {
+            desc: true,
+            id: ColumnHeaders.Volume,
+          },
+        ],
+      },
+      getTrProps,
+      ...props,
+    },
+    useSortBy
+  )
+
+  /*
+  useEffect(() => {
```

Comment:
any reason this is commented out but kept?

Reply:
still needed, just needed some changes. Its updated now.